src/parser/demonhunter/havoc/modules/talents/Demonic.tsx

```diff
@@ -63,14 +67,14 @@ class Demonic extends Analyzer {
     this.deathsweepsInMetaCounter = 0;
   }
 
-  onDeathSweepCast(event) {
+  onDeathSweepCast(event: CastEvent) {
     if (this.eyeBeamTimeStamp !== undefined && (event.timestamp - this.eyeBeamTimeStamp) < META_BUFF_DURATION_EYEBEAM) {
```

Comment:
`this.eyeBeamTimeStamp !== undefined` can never be undefined

Reply:
Fixed